src/genie/libs/parser/iosxe/show_isis.py

```diff
@@ -26,11 +27,14 @@ class ShowIsisNeighborsSchema(MetaParser):
                     Any(): {
                         'type': {
                             Any(): {
-                                'circuit_id': str,
-                                'holdtime': str,
-                                'interface': str,
-                                'ip_address': str,
-                                'state': str,
+                                'interfaces': {
```

Comment:
thanks for updating this. this makes sense. schema change looks good.

Reply:
Hi Tahigash,

I'm happy to continue contributing to the development of this awesome tool!

Thanks!
Ernesto Wilson